src/pilot/powerflex/powerflex-ansible/roles/powerflex-common/tasks/install_powerflex.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
I'm a bit confused. Aren't the powerflex-ansible files and few coming from https://github.com/dell/tripleo-powerflex/tree/master/powerflex-ansible rpm installation right. Are we packaging them with JetPack?

Reply:
Yes all is packaged and delivered with JetPack. The tripleo-powerflex repository aime to be used for standalone deployment, not using JetPack.

Reply:
> Yes all is packaged and delivered with JetPack. The tripleo-powerflex repository to be used for standalone deployment, not using JetPack.

I assume we will have to use what is distributed by RH. We are not packaging this with JetPack. The out of tree repo will be distributed by RH and it will land in RHOSP and will be available for deployment, Let's touch base next week

Reply:
Then we have to discuss since this is how the code has been written so far.

Reply:
For now we _are_ packaging everything into JetPack. We can review once out of tree repo is available through rpms ,but that s not the case as of now.

Reply:
I'm ok. If it is easier to clone let's do that instead since maintaining two repos is not preferred